docs/source/index.rst

```diff
@@ -1,33 +1,10 @@
-Snowpark API Reference (Python)
+Python API Reference
```

Comment:
We also have another, different, Python API called "Snowflake API for Python" so the proposed change could be confusing.

Reply:
What would you suggest having the title as here?

Reply:
we  should keep it as is " Snowpark API Reference (Python)"